man/io_uring_setup.2

```diff
@@ -148,6 +149,27 @@ The
 .I resv
 array must be initialized to zero.
 
+.I features
+is filled in by the kernel, which specifies various features supported
+by current kernel version.
+.TP
+.B IORING_FEAT_SINGLE_MMAP
+.TP
+.B IORING_FEAT_NODROP
+.TP
+.B IORING_FEAT_SUBMIT_STABLE
+If this flag is set, applications can be certain that any data for
+async offload has been consumed when the kernel has consumed the SQE.
+.TP
+.B IORING_FEAT_RW_CUR_POS
+If this flag is set, applications can specify offset == -1 to mean
+file position, which behaves like preadv2/pwritev2 with offset == -1,
```

Comment:
Should be:

file position, which behaves like
.I preadv2(2)
and
.I pwritev2(2)
with offset == -1,